docker/sequencer/deploy_contracts.sh

```diff
@@ -114,3 +104,14 @@ cargo run \
     deploy-proxy \
     --owner $DEVNET_ACCOUNT_ADDRESS \
     --fee 1
+
+# Deploy the dummy ERC20 contracts
+cargo run \
+    --package scripts -- \
+    --priv-key $DEVNET_PKEY \
+    --rpc-url $DEVNET_RPC_URL \
+    --deployments-path $DEPLOYMENTS_PATH \
+    deploy-erc20s \
+    --account-skeys $DEVNET_PKEY \
+    --tickers DUMMY1 DUMMY2 \
+    --funding-amount 1000
```

Comment:
Let's increase this funding amount, as mentioned in the other PR, this will not be enough if we're doing e.g. an ETH/USDC trade on a real price stream. I'd suggest an amount larger than we'll ever need